docs/reference/requirement-sets/powerpoint-api-1-2-requirement-set.md

```diff
@@ -0,0 +1,41 @@
+---
+title: PowerPoint JavaScript API requirement set 1.2
+description: 'Details about the PowerPointApi 1.2 requirement set.'
+ms.date: 12/04/2020
+ms.prod: powerpoint
+localization_priority: Normal
+---
+
+# What's new in PowerPoint JavaScript API 1.2
+
+PowerPointApi 1.2 added support for inserting slides from another presentation into the current presentation and for deleting slides.
+
+The first table provides a concise summary of the APIs, while the subsequent table gives a detailed list.
+
+| Feature area | Description | Relevant objects |
+|:--- |:--- |:--- |
+| Insert and Delete Slides | Allows the insertion of existing slides into the current presentation from another presentation, as well as the ability to delete slides. | [Slide.delete](/javascript/api/powerpoint/powerpoint.slide#delete--), [Presentation.insertSlidesFromBase64](/javascript/api/powerpoint/powerpoint.presentation#insertslidesfrombase64-base64file--options-)|
+
+## API list
+
+The following table lists the PowerPoint JavaScript API requirement set 1.2. For a complete list of all PowerPoint JavaScript APIs (including preview APIs and previously released APIs), see [all PowerPoint JavaScript APIs](/javascript/api/powerpoint?view=powerpoint-js-preview&preserve-view=true).
+
+| Class | Fields | Description |
```

Comment:
Was this hand-authored or pulled from https://github.com/OfficeDev/office-js-docs-reference/tree/master/docs/requirement-set-tables?

Reply:
@AlexJerabek I copied it from the old preview req set article which you made in October when I was out. It looks like you copied it from that file you linked to.